src/cmds/orgs:create.ts

```diff
@@ -45,11 +44,43 @@ export async function handler(argv: {
         ? `Created ${res.orgs.length} out of ${res.totalOrgs} organization(s). Written the data to file: ${res.fileName}`
         : `⚠ No organization(s) created!`;
 
-    console.log(orgsMessage);
+    return {
+      fileName: res.fileName,
+      errorMessage: undefined,
+      exitCode: 0,
+      message: orgsMessage,
+    };
   } catch (e) {
     const errorMessage = `ERROR! Failed to create organizations.\nTry running with \`DEBUG=snyk* <command> for more info\`.\nERROR: ${e.message}`;
-    debug('Failed to create organizations.\n' + e);
-    console.error(errorMessage);
-    setTimeout(() => yargs.exit(1, new Error(errorMessage)), 3000);
+    return {
+      fileName: undefined,
+      errorMessage: errorMessage,
+      exitCode: 1,
+      message: undefined,
+    };
+  }
+}
+
+export async function handler(argv: {
+  file: string;
+  includeExistingOrgsInOutput: boolean;
+  noDuplicateNames?: boolean;
+}): Promise<void> {
+  const { file, noDuplicateNames, includeExistingOrgsInOutput } = argv;
+  debug('ℹ️  Options: ' + JSON.stringify(argv));
+
+  const res = await createOrg(
+    file,
+    includeExistingOrgsInOutput,
+    noDuplicateNames,
+  );
+
+  if (res.exitCode === 1) {
+    debug('Failed to create organizations.\n' + res.errorMessage);
+
+    console.error(res.errorMessage);
+    setTimeout(() => yargs.exit(1, new Error(res.errorMessage)), 3000);
```

Comment:
What does this do?

Reply:
the set Timeout? it's because otherwise we don't have the time to write in the log file.